<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewShapeChoice</class>
 <widget class="QDialog" name="NewShapeChoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(170,170,170);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>Available shapes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makeHex">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 3px solid rgb(170, 0, 0);
background: rgb(0, 0, 0);
color: rgb(255, 0, 0);
padding: 2px;
}

QPushButton:hover{
	border: 3px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
}

QPushButton:pressed{
    border: 4px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
	font-weight:bold;
}
</string>
       </property>
       <property name="text">
        <string>Hexahedron</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makeTetra">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 3px solid rgb(170, 0, 0);
background: rgb(0, 0, 0);
color: rgb(255, 0, 0);
padding: 2px;
}

QPushButton:hover{
	border: 3px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
}

QPushButton:pressed{
    border: 4px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
	font-weight:bold;
}
</string>
       </property>
       <property name="text">
        <string>Tetrahedron</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makePyr">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 3px solid rgb(170, 0, 0);
background: rgb(0, 0, 0);
color: rgb(255, 0, 0);
padding: 2px;
}

QPushButton:hover{
	border: 3px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
}

QPushButton:pressed{
    border: 4px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
	font-weight:bold;
}
</string>
       </property>
       <property name="text">
        <string>Pyramid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makeSphere">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 3px solid rgb(170, 0, 0);
background: rgb(0, 0, 0);
color: rgb(255, 0, 0);
padding: 2px;
}

QPushButton:hover{
	border: 3px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
}

QPushButton:pressed{
    border: 4px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
	font-weight:bold;
}
</string>
       </property>
       <property name="text">
        <string>Sphere</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makeCyl">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 3px solid rgb(170, 0, 0);
background: rgb(0, 0, 0);
color: rgb(255, 0, 0);
padding: 2px;
}

QPushButton:hover{
	border: 3px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
}

QPushButton:pressed{
    border: 4px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
	font-weight:bold;
}
</string>
       </property>
       <property name="text">
        <string>Cylinder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makeCone">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 3px solid rgb(170, 0, 0);
background: rgb(0, 0, 0);
color: rgb(255, 0, 0);
padding: 2px;
}

QPushButton:hover{
	border: 3px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
}

QPushButton:pressed{
    border: 4px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
	font-weight:bold;
}
</string>
       </property>
       <property name="text">
        <string>Cone</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makeDisk">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 3px solid rgb(170, 0, 0);
background: rgb(0, 0, 0);
color: rgb(255, 0, 0);
padding: 2px;
}

QPushButton:hover{
	border: 3px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
}

QPushButton:pressed{
    border: 4px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
	font-weight:bold;
}
</string>
       </property>
       <property name="text">
        <string>Disk</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makePlane">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 3px solid rgb(170, 0, 0);
background: rgb(0, 0, 0);
color: rgb(255, 0, 0);
padding: 2px;
}

QPushButton:hover{
	border: 3px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
}

QPushButton:pressed{
    border: 4px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
	font-weight:bold;
}
</string>
       </property>
       <property name="text">
        <string>Plane</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>Other</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makePoints">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 3px solid rgb(170, 0, 0);
background: rgb(0, 0, 0);
color: rgb(255, 0, 0);
padding: 2px;
}

QPushButton:hover{
	border: 3px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
}

QPushButton:pressed{
    border: 4px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
	font-weight:bold;
}
</string>
       </property>
       <property name="text">
        <string>Point cluster</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makeLine">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 3px solid rgb(170, 0, 0);
background: rgb(0, 0, 0);
color: rgb(255, 0, 0);
padding: 2px;
}

QPushButton:hover{
	border: 3px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
}

QPushButton:pressed{
    border: 4px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
	font-weight:bold;
}
</string>
       </property>
       <property name="text">
        <string>Line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makeEarth">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 3px solid rgb(170, 0, 0);
background: rgb(0, 0, 0);
color: rgb(255, 0, 0);
padding: 2px;
}

QPushButton:hover{
	border: 3px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
}

QPushButton:pressed{
    border: 4px solid rgb(255, 255, 255);
	background: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
	padding: 2px;
	font-weight:bold;
}
</string>
       </property>
       <property name="text">
        <string>Earth</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
